Resources.php

```diff
@@ -651,6 +661,16 @@
 		'position' => 'top',
 	),
 
+	'ext.srf.gallery.lightgallery' => $formatModule + array(
+		'scripts' => 'gallery/resources/ext.srf.gallery.lightgallery.js',
+		'styles'  => 'gallery/resources/ext.srf.gallery.lightgallery.css',
+		'dependencies' => array(
+			'ext.srf.formats.gallery',
+			'ext.jquery.lightgallery'
```

Comment:
So these new files are always loaded even when the feature is not used? That's not ideal